harness/determined/exec/harness.py

```diff
@@ -163,7 +163,12 @@ def main() -> None:
     container_id = os.environ["DET_CONTAINER_ID"]
     hparams = simplejson.loads(os.environ["DET_HPARAMS"])
     initial_work = workload.Workload.from_json(simplejson.loads(os.environ["DET_INITIAL_WORKLOAD"]))
-    latest_checkpoint = simplejson.loads(os.environ["DET_LATEST_CHECKPOINT"])
+
+    # Load latest checkpoint from file to handle large data format
+    with open(os.environ["DET_LATEST_CHECKPOINT"], "r") as f:
+        latest_checkpoint = json.load(f)
+    f.close()
```

Comment:
no need for an `f.close()`, as the `__exit__()` method of the file object (which is the python magic function that is always called when the `with` block ends) will close the file for you.